src/services/posts/delete-post.test.js

```diff
@@ -0,0 +1,23 @@
+const {expect} = require('chai')
+const db = require("../../db")
+const deletePost = require("./delete-post")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('DELETE gif', () => {
+	let user
+	before(async ()=>{
+		 user = await fixtures.insertUser()  
+	})
+	it('should delete a gif', async () => {
+		
+	    const post = await fixtures.insertPost({userId : user.id})
```

Comment:
pass the `type` also here else the test will not be really accurate

Reply:
okayy but the type will be hardcoded to gif this time around right?

Reply:
yep

Reply:
done and pushed